src/stratigraphy/util/find_description.py

```diff
@@ -183,7 +189,8 @@ def _block_separated_by_line(
         line_y_coordinate = (line.start.y + line.end.y) / 2
 
         is_line_long_enough = (
-            np.min([block.rect.x1, line_right_x]) - np.max([block.rect.x0, line_left_x]) > threshold * block.rect.width
+            np.min([material_description_rect.x1, line_right_x]) - np.max([material_description_rect.x0, line_left_x])
+            > threshold * block.rect.width
```

Comment:
Don't we rather want to compare with `threshold * material_description_rect.width` here? That would also make the `block` parameter to this method redundant.

Reply:
Yep, you are right!

Reply:
Your suggested change increased F1 to 86.7%. Thank you!